docs/cql/ddl.rst

```diff
@@ -289,6 +289,16 @@ For instance::
 
 The supported options are the same as :ref:`creating a keyspace <create-keyspace-statement>`.
 
+ALTER tablets-enabled KEYSPACE :label-caution:`Experimental`
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Modifying a tablets-enabled keyspace is possible and there is no special CQL syntax introduced for that purpose, because in case a keyspace has tablets, altering it follows the same process as with vnode keyspaces, though there are some caveats:
```

Comment:
s/caveats/limitations/

Reply:
@annastuchlik please review too

Reply:
> and avoid mentioning VNodes

`docs/*.rst` do mention "vnode" on many occasions, hence I mentioned it as well. As far as I understand, in this section we want to present differences of processing a vnode vs tablets KS. That being said, I can make it shorter and avoid vnode reference, just let me know what's the final decision here.

Reply:
Applied suggestion:
> Modifying a keyspace with tablets enabled is possible and doesn't require any special CQL syntax. However, there are some limitations: